pyre/extra_tests/parity_tests/simplequeue_put_defaults_replay.py

```
@@ -0,0 +1,73 @@
+
+The count is small enough to stay fast and large enough for the loop to be
+compiled and left at least once.
+"""
+
+import queue
+
+LIMIT = 1500
+
+
+class Counter:
+    def __init__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)